corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -231,6 +231,19 @@
             </button>
           </form>
         {% endif %}
+        {% if is_archived %}
+          <div style="display: inline-block;">
```

Comment:
I think if you put this in a `span` instead of a `div` you can drop the inline style, which is borderline verboten.

Reply:
`span` didn't work, but `form` did. I had `form` initially but also had a submit action defined and was seeing an artifact when trying displaying the module, so I scrapped it but should have just removed all of the cruff. See a452a911d9a348e3a45bc6eb7746ed6b26650db1